pkg/sql/information_schema.go

```diff
@@ -460,6 +461,36 @@ https://www.postgresql.org/docs/9.5/infoschema-columns.html`,
 	},
 }
 
+var informationSchemaColumnUDTUsage = virtualSchemaTable{
+	comment: `columns with user defined types
+` + base.DocsURL("information-schema.html#column_udt_usage") + `
+https://www.postgresql.org/docs/current/infoschema-column-udt-usage.html`,
+	schema: vtable.InformationSchemaColumnUDTUsage,
+	populate: func(ctx context.Context, p *planner, dbContext *dbdesc.Immutable, addRow func(...tree.Datum) error) error {
+		return forEachTableDesc(ctx, p, dbContext, hideVirtual,
+			func(db *dbdesc.Immutable, scName string, table catalog.TableDescriptor) error {
+				dbNameStr := tree.NewDString(db.GetName())
+				scNameStr := tree.NewDString(scName)
+				tbNameStr := tree.NewDString(table.GetName())
+				return table.ForeachPublicColumn(func(col *descpb.ColumnDescriptor) error {
+					if !col.Type.UserDefined() {
```

Comment:
/chime
looks like postgres builtins are "user defined" too?
```
otan        | pg_catalog | oid             | otan          | pg_catalog         | pg_proc                               | oid
 otan        | pg_catalog | name            | otan          | pg_catalog         | pg_proc                               | proname
 otan        | pg_catalog | oid             | otan          | pg_catalog         | pg_proc                               | pronamespace
 otan        | pg_catalog | oid             | otan          | pg_catalog         | pg_proc                               | proowner
 otan        | pg_catalog | oid             | otan          | pg_catalog         | pg_proc                               | prolang
 otan        | pg_catalog | float4          | otan          | pg_catalog         | pg_proc                               | procost
 otan        | pg_catalog | float4          | otan          | pg_catalog         | pg_proc                               | prorows
 otan        | pg_catalog | oid             | otan          | pg_catalog         | pg_proc                               | provariadic

```
/chime

Reply:
Yeah this is on the docs, but postgres mentions that they only include them because of how the implementation is. I think it's fine (and better) for us to not include the builtins, because then this table is the same as the existing `columns` table.